packages/common/keys/src/space-id.ts

```diff
@@ -0,0 +1,39 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import base32Decode from 'base32-decode';
+import base32Encode from 'base32-encode';
+
+import { invariant } from '@dxos/invariant';
+
+import { randomBytes } from './random-bytes';
+
+/**
+ * A unique identifier for a space.
+ */
+export type SpaceId = string & { __SpaceId: true };
```

Comment:
make __SypaceID a symbol (and everywhere else we need this pattern?)

Reply:
I think a string tag is simpler and more then enough for this usecase. automerge uses them - https://github.com/automerge/automerge-repo/blob/main/packages/automerge-repo/src/types.ts#L5

I'm not really sure how symbols would behave here